docker-compose.yml

```diff
@@ -15,6 +15,7 @@ x-sentry-defaults: &sentry_defaults
     - smtp
     - snuba-api
     - snuba-consumer
+    - snuba-outcomes-consumer
```

Comment:
Do we really need this ?
This will force snuba-outcomes-consumer service to be running for any of the: web, cron, worker, event-consumer, post-process-forwarder and sentry-cleanup.

Reply:
Generally speaking we should run it. I am not sure about how our docker-compose is structured, but it seems like we only have two dependency lists, one for any snuba container and one for any sentry container.

Reply:
@untitaker we can have a dependency list for any service so if this can be a dependency of something else, like relay we can put it there too.

Reply:
It's up to you how granular you want this to be, but I can't imagine this consumer being optional in any setup